slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c

```diff
@@ -1503,7 +1503,7 @@ int set_module(int file_des) {
     ret = OK;
     memset(mess, 0, sizeof(mess));
 
-#if defined(CHIPTESTBOARDD) || defined(MOENCHD) || defined(GOTTHARD2D)
+#if !(defined(MYTHEN3D) && defined(EIGERD))
```

Comment:
Should probably be an || ? As it stands now both Mythen3 and Eiger has to be defined...

Reply:
Thanks!